Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -2375,7 +2375,7 @@
 					"version": 1,
 					"id": "feff8222-53e6-431e-b946-0d13bdf135e4",
 					"disabled": true,
-					"name": "Only Damaged by @Burning/Cutting/Crushing@ or @Burning/Cutting/Crushing@",
+					"name": "Only Damaged by @Burning/Corrosion/Cutting/Crushing@ or @Burning/Corrosion/Cutting/Crushing@",
```

Comment:
This (and similar one(s) below) will not give you what you want. When the dialog comes up to prompt the user for substitutions, it goes through and finds all of them attached to a given object and then only presents the unique ones. So in this case, it would present a field with a title of "Burning/Corrosion/Cutting/Crushing" once, the user would fill it in, then it would replace both spots with what the user provided (i.e the same thing in both). To get different things in each, the content between the @ symbols must be different in some way. I'd suggest something like:

Only damaged by @1. Choose one of Burning, Corrosion, Cutting or Crushing@ or @2. Choose one of Burning, Corrosion, Cutting or Crushing@

Reply:
Yep.. now I see what you're saying.